### PR TITLE
qemu/qemu_v8: synchronise build scripts

### PR DESCRIPTION
**Pull request by etienne-lms**
Following https://github.com/OP-TEE/build/pull/252 and https://github.com/OP-TEE/build/pull/254, this P-R synchronises the script layout of **qemu.mk** and **qemu_v8.mk**.

Commit comment:
```
Qemu and qemu_v8 are almost built the same way. Main differences are
32b/64b modes and bios_qemu_tz_arm/edk2 as non-secure boot stage.

optee-client targets are removed since now built from the buildroot
target.

This change allows enables SMP testing on qemu_v8 through the QEMU_SMP
directive.
````

### CODE REVIEW

File: qemu_v8.mk [hunk 1]
```diff
@@ -40,27 +37,37 @@ include toolchain.mk
...
-	ARM_TSP_RAM_LOCATION=tdram \
 	PLAT=qemu \
-	DEBUG=0 \
-	LOG_LEVEL=50 \
+	ARM_TSP_RAM_LOCATION=tdram \
 	BL32_RAM_LOCATION=tdram \
-	SPD=opteed
+	SPD=opteed \
+	DEBUG=0 \
+	LOG_LEVEL=50
```
Comment (etienne-lms) [inline qemu_v8.mk]: `$(ARM_TF_DEBUG)` and `$(ARM_TF_LOGLVL)`

Comment (jenswi-linaro) [reply qemu_v8.mk]: Yes, those flags should be used.

File: qemu_v8.mk [hunk 2]
```diff
@@ -109,6 +114,8 @@ linux-defconfig: $(LINUX_PATH)/.config
 LINUX_COMMON_FLAGS += ARCH=arm64
 
 linux: linux-common
+	mkdir -p $(BINARIES_PATH)
+	ln -sf $(LINUX_PATH)/arch/arm/boot/Image $(BINARIES_PATH)
```
Comment (etienne-lms) [inline qemu_v8.mk]: Bug: should be **arch/arm64/boot/Image**

**Comment by etienne-lms** [comment]
squashed. tag not applied yet (few changes since jens' tag)

**Comment by jbech-linaro** [comment]
@etienne-lms , kill this PR since it touches things no longer relevant etc?

**Comment by etienne-lms** [comment]
Hi, I had a look. Most change would still apply. I will send an update.

**Comment by etienne-lms** [comment]
rebased.

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by etienne-lms** [comment]
tag applied

**Pull request merged**